src/main/resources/public/js/graphs.controller.js

```diff
@@ -121,7 +121,9 @@
           if (multipleSeries) {
             vm.distanceSeriesLabels.push(year);
             for (const monthKey in vm.distanceLabels) {
-              series.push(data[vm.distanceLabels[monthKey]] || 0);
+              let valueForMonth = data[vm.distanceLabels[monthKey]];
+              valueForMonth = !isNaN(valueForMonth) ? parseFloat(valueForMonth).toFixed(2) : 0;
```

Comment:
Maybe a workaround bit still work to prevent a too long distance in the tooltip.